packages/realm-react/example/android/app/src/main/java/com/tsexample/MainActivity.java

```diff
@@ -0,0 +1,15 @@
+package com.tsexample;
+
+import com.facebook.react.ReactActivity;
+
+public class MainActivity extends ReactActivity {
+
+  /**
+   * Returns the name of the main component registered from JavaScript. This is used to schedule
+   * rendering of the component.
+   */
+  @Override
+  protected String getMainComponentName() {
+    return "TsExample";
```

Comment:
I should rename this to RealmReactExample.  It will be easier to just reinitialize the entire example then, as this is a very complicated matter to change the name by hand

Reply:
done